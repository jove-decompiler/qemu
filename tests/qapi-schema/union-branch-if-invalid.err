tests/qapi-schema/union-branch-if-invalid.json: In union 'Uni':
tests/qapi-schema/union-branch-if-invalid.json:4: 'if' condition '' makes no sense
