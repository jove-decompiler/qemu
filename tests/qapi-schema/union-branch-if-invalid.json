# Cover branch with invalid 'if'
# FIXME not rejected, would generate '#if \n'
{ 'enum': 'Branches', 'data': ['branch1'] }
{ 'struct': 'Stru', 'data': { 'member': 'str' } }
{ 'union': 'Uni',
  'base': { 'tag': 'Branches' }, 'discriminator': 'tag',
  'data': { 'branch1': { 'type': 'Stru', 'if': [''] } } }
