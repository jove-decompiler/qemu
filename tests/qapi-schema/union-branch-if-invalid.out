module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module union-branch-if-invalid.json
enum Branches
    member branch1
object Stru
    member member: str optional=False
object q_obj_Uni-base
    member tag: Branches optional=False
object Uni
    base q_obj_Uni-base
    tag tag
    case branch1: Stru
        if ['']
